packages/vue/src/components/molecules/SfAddToCart/SfAddToCart.html

```diff
@@ -0,0 +1,16 @@
+<div class="sf-add-to-cart" data-test="sf-add-to-card">
+  <SfButton v-on="$listeners" class="sf-add-to-cart__button">
+    <!--@slot Custom content that will replace default Add to cart button design.-->
+    <slot name="add-to-cart-btn">
```

Comment:
```suggestion
    <slot name="add-to-cart-btn-content">
```
Maybe something like this?

Reply:
right. Will change :)